Maxbounds on Map props

SBRK opened the pull request.

alex3165 commented on src/map.tsx:
```diff
@@ -264,7 +265,7 @@ const ReactMapboxFactory = ({
         minZoom,
         maxZoom,
-        maxBounds,
+        maxBounds: this.props.maxBounds || maxBounds,
```
Can you remove ` || maxBounds` and remove it from the factory parameter (line 197), also destruct `maxBounds` from `this.props` (line 25*)

alex3165 commented:
Nice job, I left a comment, could you update the documentation accordingly?

SBRK commented:
@alex3165 I updated the doc and removed maxBounds from the factory, but the tests are failing now and I have no idea why (because I see `maxBounds` nowhere on the tests, so I don't know what's going on)

alex3165 commented:
Look like tests are passing on the CI, I will merge the PR and check on master. Nice job 👍

SBRK commented:
Yeah, I forgot to `npm i` before building :)

alex3165 commented:
Released with version `2.4.0`
